Unreal Launcher/Unreal Launcher/MainWindow.xaml.cs

```diff
@@ -40,7 +40,7 @@ public MainWindow()
 
 			if (Settings.Default.Projects != null)
 			{
-				System.Collections.Specialized.StringCollection InvalidProjects = new System.Collections.Specialized.StringCollection();
+				System.Collections.Specialized.StringCollection invalidProjects = new System.Collections.Specialized.StringCollection();
```

Comment:
This would probably be nicer if you imported the `System.Collections.Specialized` namespace